llm_on_ray/inference/models/deepseek-coder-33b-instruct.yaml

```diff
@@ -0,0 +1,25 @@
+port: 8000
+name: deepseek-coder-33b-instruct
+route_prefix: /deepseek-coder-33b-instruct
+num_replicas: 1
+cpus_per_worker: 24
+gpus_per_worker: 0
+deepspeed: false
+workers_per_group: 2
+device: "cpu"
```

Comment:
```suggestion
device: cpu
```

Reply:
The latency of this model significantly exceeds that of the 7b model; however, it outperforms other models, including Codellama, in the specific task of converting Scala code to Velox UDF.